recipes/intake-parquet/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.2.1" %}
+{% set name = "intake-parquet" %}
+{% set sha256 = "4d72e8faf2c3ca2f9a8311ffc73510040e764949b7706f44187895b278ca64be" %}
+
+package:
+   name: {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
+  run:
+    - python
+    - intake>=0.3
```

Comment:
I am being pedantic, but it would be great to add a space here, e.g. `intake >=3`.